build.gradle

```diff
@@ -670,6 +692,7 @@ dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.7.2'
     testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.7.2'
     testCompileOnly 'junit:junit:4.13.2'
+    implementation group: 'org.javassist', name: 'javassist', version:'3.28.0-GA'
```

Comment:
Did you figure out why we suddenly needed this?

Reply:
I didn't. The 1.3 CI started failing due to missing javassit since Oct 5.